build.gradle

```diff
@@ -142,6 +144,17 @@ task docker(type: Docker) {
     entryPoint(['java', '-jar', "/${project.name}/${project.name}.jar"])
 }
 
+testprofiler {
+    minTestThreshold = 100
+
+    buildBreaker {
+        maxTestThreshold = project.hasProperty('maxTestThreshold') ? project.property('maxTestThreshold') : 30_000

```

Comment:
I have to contribute to Gradle something like:
`getPropertyIsExistOrReturnDefaultValue('maxTestThreshold', 30_000)`